clean_cost_report_1.py

```diff
@@ -0,0 +1,167 @@
+#####define function for extracting a given list of desired variables from the raw dataset
+
+def cost_data_reduce(costdata, varlist):
+    costdata['varname'] = ''
+    line_select = pd.Series(np.zeros(len(costdata))).astype(int)
+    for i in range(len(varlist[0])):
+        new_line_select = ((costdata['wksht_cd']==varlist[0][i]) & (costdata['line_num']==varlist[1][i]) & \
```

Comment:
Two issues:
1) don't stick three boolean conditions in a single statement; if you need to evaluate three booleans, assign each to a named variable and then check the named variables (I made up variable names here; not sure if they're right/useful given the code's semantics):
```
is_wksht = costdata['wksht_cd']==varlist[0][i]
is_line_num = costdata['line_num']==varlist[1][i]
is_col_num = costdata['clmn_num']==varlist[2][i]
new_line_select = is_wksht & is_line_num & is_col_num
```
2) don't use `&`. You got lucky in this case that it behaves correctly, but in general if you want a logical `and`, Python lets you just say `and`. The `&` operator is a bit-wise and. Booleans are a single bit so it behaves the same as `and`, but it will fail on numbers, strings and objects, etc. The semantics you want are logical `and`, so just use that word.